lib/file.js

```diff
@@ -210,22 +210,26 @@ internals.openStat = function (path, mode, callback) {
             }
 
             if (err.code === 'EACCES' || err.code === 'EPERM') {
+                return callback(Boom.forbidden(null, err.code));
+            }
+
+            if (path.indexOf('\u0000') !== -1) {      // handle "Path must be a string without null bytes" error
```

Comment:
I placed it where it is, since it will only be called for what would otherwise be a internal error. This way it won't have any impact on normal responses.